x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/rule_alerts_summary.tsx

```diff
@@ -82,12 +114,20 @@ export const RuleAlertsSummary = ({ rule, filteredRuleTypes }: RuleAlertsSummary
                   />
                 </h5>
               </EuiTitle>
+              <EuiSpacer size="s" />
```

Comment:
@maciejforcone I am wondering if `xs` will look better for `EuiSpacer` and `EuiText`

Reply:
@XavierM could you share the screenshot? I'm now sure which title is S. I'd like to have the same font-sizes as in rule summary box next to alerts widget. Same title style, same second-line timeframe style etc.

Reply:
@XavierM, for the `EuiText` in the rule_status_panel (left panel in the rule details page). the title is smaller than the message under it. 
And the spacer is also `s`


<img width="926" alt="Screenshot 2022-07-29 at 14 54 28" src="https://user-images.githubusercontent.com/6838659/181764867-a7e1b3ee-6deb-4258-84ac-e0a2f2c9e4a3.png">

Reply:
it is matching like you said @fkanout and @maciejforcone 

<img width="739" alt="image" src="https://user-images.githubusercontent.com/189600/181766760-a6574699-7e42-4405-8cbb-bd4890c7df90.png">